competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2516 @@
l1I1UZiJ4jyfTnTuy0jRVIeOTLPPhBwabS4kwjcR5FhgmooE83p+Y54guHYVR5BBkjlhYVFbM40DX9fTdSOG0dIxZk1QUwe04CZZMKYauJ+XMatVSNw3HzhNoePqjf43efUea3pJyTVVCreGX394xjImnu5qcFDerhvvJcxoH1pNjngPOGKLSRBS2MGAdo+84TYEQEoXSKGdwRqHijI9axkPWorX8npP3FCaRfSIGUSyMKRC1om1bCDP1EoKeRClD4bSQgBScu47SR1ZtRc4wTYGYEnVTUi6BSroqiIvTKclGjf3dmayhdJph9Kzbhk37AFz4fo+/sdgpaeH+V8Cf5pz/3Y/+6v8M/A+Bf2f55//poz//3yql/l1kQfG7wH/y130PbTSubDCugCywPmtqVusrysPItim53Ws8PIqOdYYY5W2ttEJUXhkfA9YohuC57QbWTcmzqzXH88Dp2JGyMOBURjJDl4tCLg8JblRKP15QKmeSyo8AgOAzIS3aLhb5yuL1REmxjCmhFj2eNYY+BNCaalFkmKS4P0/MC704hsToA7W1zHNgjoJ62jQlb889392fqZxj9Iln65r3/biE0SxFH1hXjqtVjbGKdVkyho6LumLX1vgoyPvRB/7y3Z5h9vzB80uKpaMstGxg8zITNUpxOA1Yq8XFsGyHSycbWynwmrQcH6WdEhS8XZYvKbNQoKUISQ6GcOj0kvZeWhHz5uzFE12sqNoSkyO+68jVDrXotJTSGAU5+MU2Z0gY4lJoFJEcAyn6JQvWLBtyyQlOSWZLoJfjlyLFHmuhqgyhmyDzOLtCK86zZN6WTrbKzhoWRZTM1wyYsiAbvRjfveg3yWxKx7nrMdZQF46UM93hyDxOaBIGcdkkNKtVjY8BrSzKFPikaMpSjubGEXxGlyXZNfzO51dcXLa8+abmy6ZiPB5ojehKf/nqjn5e47RmvVrxB1/uuNmW/OM//5Z9N7KpHa9uD1xu16he5tCmcGg019sacmJdV5zGiO8Dla0wNmFdAchiLvmJQonn2FqFK0rS5NEhMIcR5yyFKwlZEf1MoRU+aowytE3B88IJJNVH+nmmKR39KGoEqxNNUWOKApUyUYlgf7OpuT+ewWhSTjRNxenYsd42vL09/k0l7PHxfTq7/wbwPwD+qVLqnyx/9r9Eity/r5T6HwNfAf89qRv5j5VS/z7wJ8gm93/y121iAZQSIabWUrIEoGkpmhXrpuBqu+bbd3tCkiNoylKWZDsjHsQYlgwIDVkLOv08jhz6get1y2YjQb5msWrlvDDytBKO+nJBpZwf518sm9Ychbogq4xlLLQM25VaCq56sLwlLBqXE3NCkEtkisIw5UyIiUOO6ElhreazzYaUEqOPPL1oOQ0zt73kRrx/N7JdVeKe0ErIEVrz02eX/OzdnnGOIqTWiqumYttW3Gwbjv3EeQrcrFqUgl++PYiMoiyZcmRXVew2a37+6j1NWRCMIyzl3ugl6DspoheZT+ks4xwoli41xYQzVgKS0gc9nWgZP8gwUhYaTYwC7GyqRTdlFJVROK3ww4wvOzIDleuoCWQKxqBpNBTOgjJonZfCqUlhQLlqyQmWVyQHTwyzzHqMRRuZ5aQUFqlQRmsj0gljwWimaWn+YiQlCWdS2mCW5/QwTmwbQZqjlERNqoxLErhjraZqSuZhFHQ2UDrL2+PAthEvJ96T54kQg8ApQ+A8zgzesG1KBjJFYcVZgyEojTFSxEOMWJ0xRYsxJbooaR2UG0P97JJvh1u+UpqzD1xuVvRj4Odv97RVybOLFXf3ZwzS+Q7ei74uJF6/v2e9qmQkozW6rFDWynNpNEknKqtYrQsSZpmReyojN5o5JuZxZmsqsp/RKWCMxuflGtLVcrwUbFfQpaTFpUQOs8RgxkBhnYT0LOOOYY5UVRQHjNFM84jV4mS52K7x3nPoetYL6HScpn+5ouKc8/+b3z6HA/jv/HM+598G/u3v/VMo6bKUFldEUtI+K1PR1C273Y66fCM5p0oCiEOSTsF7MXErtRxR9cPcDZqiWI6qmV1b88YaiAmdhVKRkAtUayMOhuW3jCkLMHTZwD1sFhWSQ6GUhHZro4lB+Pp6OVrLjyBDc6syCdERXtcF+34ma2HD5SjRjoVS6LLAB9ly3adMzJnrpma2gcoZgs9MQbq9t8ee0zgzDMLTezhcjyHSz5F9N1FXBZ9fbhh84DTPfP70glM3Mk4RmyXp7Ou7A7/36VPuD2fMpWF/Gnh/39HPH4jMCjmyjrNn8kEsWkq2qT4EcbCQsUgRzFk667oomENkmr2ETy9iaWc0VoNjSSxzNcEYTrFgPpyY3n6FXm2wzRWUKxQZQ0SrJEb/ZY44+YhjAGvIWbquOM+EECT9ymSsk1cjer/4kuUGqrVsTJUqUPUNxIHgI90UacqHjl5ufo1zEvqUZKYXs6Ip9HKT1AQfOJ16vnu7l5AeZ8lKsaoL5piwMTMazf7YEbzn3b6jKQwaqKzM7FRCdGRE5nHAlJCt3GicAYxGFzVtW6NVQqVBoBTrNbsXv8cxveLY9RzDwGZrOPvXnIeRvTUM3Ugms6pKKmd4tz9zvVmxW5BNRVVSlTu220TwI/v9iWwc1xcNfpyYhoF6wVnNpz0hiw87RgXlDnRmmGayNtSlpW3El+utXGO2doDBuUzKHu89yXtBeZXLnFdDVRVLJy6JZaiRUz+S0TRtwziM+BA5dyNBK4zRuKLGWcPd4fS9y8wPwkGxjLYXBb6ENCcWGUO1Yq4uMGUJ/bwkiFmMCqiU5dikPqg6DQqLGO8FwS4m55wzxkkxJS/e1PQwNNcS66ce9q2LbAJFRIPKS7yhQms5x+gFGOoTGCOik5zzYp3KrBQ0q5JziLROohfnmJlSZm01VamJPvPV7YmsFavC4MyK2cclwctx2VbcnnpiznTjvMg2YPASPqONekQ7TT7y7f6EMSsO/cgX11sGHwg5cb1b84u3e5G85IzPiTBOqCxSmEo
```

Comment:
I think we want

```
        self.save_hyperparameters(ignore=["x_train", "y_train", "x_val", "y_val"])
```

unless we want to include the training / validation data in the hyperparameters.

Reply:
I don't think they're included as is. It only stores the arguments under `self.hparams`, which in this case doesn't include the input data - `x_train`, `y_train`, `x_val`, and  `y_val` are all input directly to `CloudModel` rather than being included in the `hparams` dict.